lib/redux/home/discover/discover_state.dart

```diff
@@ -36,10 +37,10 @@ class DiscoverState {
       other is DiscoverState &&
           runtimeType == other.runtimeType &&
           loadingStatus == other.loadingStatus &&
-          articleList == other.articleList &&
+          articlesDirectory == other.articlesDirectory &&
           selectedArticle == other.selectedArticle;
 
   @override
   int get hashCode =>
-      loadingStatus.hashCode ^ articleList.hashCode ^ selectedArticle.hashCode;
+      loadingStatus.hashCode ^ articlesDirectory.hashCode ^ selectedArticle.hashCode;
```

Comment:
maybe I need a lesson on what's happening here, if someone could explain in a coffee break.